pandas/tslib.pyx

```diff
@@ -24,6 +24,11 @@ from cpython cimport (
     PyUnicode_AsUTF8String,
 )
 
+
+cdef extern from "headers/stdint.h":
+    enum: INT64_MAX
+    enum: INT64_MIN
+
```

Comment:
This is how we do it in `lib.pyx`, but agreed it probably makes more sense to pull it from the numpy headers.